.github/actions/authenticate_to_cluster/action.yml

```diff
@@ -0,0 +1,32 @@
+name: "Authenticate to the kubernetes cluster"
+description: 'Authenticate to the kubernetes cluster'
+inputs:
+  kube-cert:
+    description: "Kubernetes cluster authentication certificate"
+    required: true
+  kube-token:
+    description: "Kubernetes cluster authentication token"
+    required: true
```

Comment:
Are these both secrets?

If they are, then we should probably be using `secrets` rather than `inputs`.

That would mean using a [reusable workflow](https://docs.github.com/en/actions/using-workflows/reusing-workflows#using-inputs-and-secrets-in-a-reusable-workflow) rather than a composite action, though.

Reply:
They are secrets but aside from this [discussion](https://github.com/orgs/community/discussions/34212) that implies it is safe to do this, there are quite a few actions  out there that rely on this behaviour, [for example](https://github.com/aws-actions/configure-aws-credentials/blob/main/action.yml#L17-L22). if the values being passed in via the workflow are from a secret "context" `${{ secret.KUBE_UAT_CERT }}` then GHA filters them out of logs. 

I have however tried to contact github to ask the question directly.

Reply:
I wasn't sure if it was clever enough to filter inputs that came from the secrets context —but that's reassuring.

To be honest I'm not sure when it's best to use a composite action or a reusable workflow (nor am I convinced GitHub do either), so I don't think it matters too much 😆